docs/src/introduction/overview.md

```diff
@@ -134,139 +170,64 @@ fuel-core --db-type in-memory
 
 This starts a Fuel node with a volatile database that will be cleared when shut down (good for testing purposes).
 
-### Deploy `wallet_contract` To Your Local Fuel Node
+### Deploy `counter_contract` To Your Local Fuel Node
 
-To deploy `wallet_contract` on your local Fuel node, run
+To deploy `counter_contract` on your local Fuel node, open a new terminal tab and run the following command from the root of the `wallet_contract` directory:
 
 ```sh
 forc deploy
 ```
 
-from the root of the `wallet_contract` directory.
+> Note you can't use the same terminal session that is running fuel-core to run any other commands as this will end your fuel-core process.
 
 This should produce some output in `stdout` that looks like this:
 
 ```console
 $ forc deploy
-  Compiled library "wallet_lib".
-  Compiled contract "wallet_contract".
-  Bytecode size is 212 bytes.
-Contract id: 0xf4b63e0e09cb72762cec18a6123a9fb5bd501b87141fac5835d80f5162505c38
+  Compiled library "core".
+  Compiled library "std".
+  Compiled contract "counter_contract".
+  Bytecode size is 208 bytes.
+Contract id: 0x1d64105ed60f22f3def36ebbda45d58513e69bcbc4b2fcce0875898b0468d276
 Logs:
-HexString256(HexFormatted(0xd9240bc439834bc6afc3f334abf285b3b733560b63d7ce1eb53afa8981984af7))
+TransactionId(HexFormatted(69a1c45f31892f61ae6b67edd8524550769b1432b7f1984ca0a456ea0de18da7))
 ```
 
-Note the contract ID—you will need it in the next step.
-
-## Write a Sway Script to Call a Sway Contract
+Note the contract ID—you will need it if you want to build out a frontend to interact with this contract.
 
-> **Note**: If you are using the SDK you do not need to write a script to call the Sway contract, this is all handled automagically by the SDK.
+## Testing your Contract
 
-Now that we have deployed our wallet contract, we need to actually _call_ our contract. We can do this by calling the contract from a script.
+In the directory `tests`, navigate to `harness.rs.` Here you'll see there is some boilerplate code to help you start interacting with and testing your contract.
```

Comment:
```suggestion
In the directory `tests`, navigate to `harness.rs`. Here you'll see there is some boilerplate code to help you start interacting with and testing your contract.
```

Reply:
unfortunately the grammatical rules of `"` don't apply to ` :(

Reply:
lolz english